README.md

```
@@ -8,16 +8,113 @@
+
+The coverage data is intersected against the provided bed file using (bedtools intserect)[bedtools-intersect-url] to identify the coverage in all provided regions, and then the binned coverage data unbinned to get per base data.
+
+This data is then summarised into the coverage per transcript and per region (i.e exons / introns) against the provided `--thresholds` and `--minimum` cut off to define as low coverage. Optional plots of the low covered regions, full genes and whole chromosomes may also be included.
+
+## :page_facing_up: Inputs
 
-## DNAnexus
 
 
 
+## <img src="athena/data/images/moby.png" width="34"/> Docker
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)